app-emulation/wa-linux-agent/wa-linux-agent-2.3.1.1.ebuild

```
@@ -12,15 +12,23 @@ LICENSE="Apache-2.0"
 SLOT="0"
 IUSE=""
 
-RDEPEND="dev-lang/python-oem"
+RDEPEND="
+dev-lang/python-oem
```

```suggestion
dev-lang/python-oem:3.6
```
so that the "remember to update" comment becomes a build time failure.

---

That's a good idea, I'll test it locally.

---

That's a neat trick and it works like a charm, thanks! I actually have put `3.6` into the `DEP_PYVER` variable, so I only need to change the number in one place in ebuild.